src/services/domain/decision.service.ts

```
@@ -0,0 +1,183 @@
+    clientProjectRoot: string,
+    repositoryName: string,
+    branch: string,
+  ): Promise<Decision[]> {
+    const logger = mcpContext.logger || console;
+    this.validateRepositoryProvider('getAllDecisions');
+
+    try {
+      const decisionRepo = this.repositoryProvider.getDecisionRepository(clientProjectRoot);
+      // Note: This method may need to be implemented in DecisionRepository
+      // For now, we'll return an empty array as a placeholder
+      logger.warn('[DecisionService.getAllDecisions] Method not fully implemented - returning empty array');
```

ESLint

[error] Replace `'[DecisionService.getAllDecisions]·Method·not·fully·implemented·-·returning·empty·array'` with `⏎········'[DecisionService.getAllDecisions]·Method·not·fully·implemented·-·returning·empty·array',⏎······`

(prettier/prettier)